crates/bevy_ecs/src/observer/trigger_targets.rs

```diff
@@ -1,117 +0,0 @@
-//! Stores the [`TriggerTargets`] trait.
```

Comment:
Yay! I am very glad that we removed this complexity. It was quite hard for users to read and reason about.